tests/006-test-overlay-cleanup.sh

```diff
@@ -22,20 +22,19 @@ EOF
  $DSSBIN --reset >> $LOGS 2>&1
  if [ $? -ne 0 ]; then
     # Test failed.
+    echo "ERROR: $testname: $DSSBIN --reset Failed." >> $LOGS
     test_status=1
  elif [ -e /etc/sysconfig/docker-storage ]; then
     # Test failed.
+    echo "ERROR: $testname: $DSSBIN --reset Failed to create /etc/sysconfig/docker." >> $LOGS
```

Comment:
This should be `cleanup` instead of `create` ? 

I wished more cleanups we had taken in another PR. More we add in single PR, more comments and rework it generates.